analytics/tests/test_counts.py

```diff
@@ -74,6 +83,10 @@
     is_cross_realm_bot_email,
 )
 
+if settings.ZILENCER_ENABLED:
+    from zilencer.models import RemoteInstallationCount, RemotePushDeviceToken, RemoteZulipServer
+    from zilencer.views import get_last_id_from_server
```

Comment:
I think these can be normal imports; we don't support running the test suite with that setting disabled.